apps/src/lab2/types.ts

```diff
@@ -209,6 +209,7 @@ export interface LevelProperties {
   miniApp?: string;
   serializedMaze?: MazeCell[][];
   startDirection?: number;
+  exemplarSettings?: ExemplarSettings;
```

Comment:
nit: put this alongside `exemplarSources` so it's near the `// Exemplars` section? maybe a small comment too (indicating that we expect there to be an exemplar source if these settings are turned on)